SolutionInfo.cs

```diff
@@ -34,3 +34,6 @@
 [assembly: AssemblyProduct("http://www.dnnsoftware.com")]
 [assembly: AssemblyCopyright("DotNetNuke is copyright 2002-2018 by DNN Corporation. All Rights Reserved.")]
 [assembly: AssemblyTrademark("DNN")]
+[assembly: AssemblyVersion("9.4.2.0")]
+[assembly: AssemblyFileVersion("9.4.2.5782")]
+[assembly: AssemblyInformationalVersion("9.4.2-unstable.5782+Branch.reorg1.Sha.b37c5e45ed0dde233bee7c8572d70940e5bda93e")]
```

Comment:
Is this supposed to be added in here?  Or when the manifests are reverted, should this be reverted as well?

Reply:
Good catch. I'll repair after merge in a fresh PR.